tutorials/06_Advanced_ChangeInputs.md

```diff
@@ -1,68 +1,77 @@
 
-Adding New Input Data
-======================
+3. Start the script with `sbatch slurm_start.sh`.
+The .log file lists the progress and potential errors. This process might take a while (currently >8 hours).
 
-The input data for REMIND are generated by using the R-libraries *mrremind* (https://github.com/pik-piam/mrremind) and *madrat* (https://github.com/pik-piam/madrat). While *mrremind* contains all calculations tailored to REMIND-input-data, the package *madrat* provides the general wrapper functions and helpful tools. For further information read the vignettes of these R-packages.
+4. If the process terminates without errors, do a test run with the new input data. To do this, clone the REMIND repo and update the data input version `cfg$revision` in `config/default.cfg` using your recently created data revision number file and run one scenario (e.g. SSP2EU-Base).
 
-How to Update Input Data
-========================
+5. If the test run completes without errors, add the change in `config/default.cfg` and the update of the input data revision in `main.gms` that was automatically performed by the REMIND run to a commit in your REMIND clone. This can be best done by using
+```bash
+git add -p config/default.cfg main.gms
+```
+and then selecting the change in `default.cfg` and the first change to the input data in `main.gms` with `y`, and then ignoring possible other changes in `main.gms` with `d`. Create a pull request to push this change to the main REMIND repository, and REMIND will use the new data by default.
 
-1. Check for the current input data revision number in this cluster folder: `/p/projects/rd3mod/inputdata/cache`. Alternatively, run the helper tool `lastRev` (`/p/projects/rd3mod/tools/lastrev`) to get a list of the last five revX.XXX*_remind.tgz items in the default moinput output directory.
-2. Clone the repo [`https://gitlab.pik-potsdam.de/REMIND/preprocessing-remind`](https://gitlab.pik-potsdam.de/REMIND/preprocessing-remind) to your tmp folder on the cluster and edit its `start.R` file by inserting the next revision number. Use at least 4 decimal places for development/testing. If an old revision number is used, the input data will not be recalculated. Input data for a new regional resolution will be recalculated based on the existing cache information.
+## Initial Parameter Values
 
-3. Start the script with
+The initial values for all optimization parameters are taken from a previous run and supplied via an input `gdx` file.
+In a bundle of runs, a previous run from the same bundle is usually configured via the `path_gdx` configuration parameter.
```

Comment:
we also use the input.gdx file in a bundle of runs. We only use information from former scenarios for time steps before startyear. I would delete this sentence here